docs/root/operations/admin.rst

```diff
@@ -307,16 +307,29 @@ modify different aspects of the server:
 
   Enable/disable logging levels for different loggers.
 
-  - To change the logging level across all loggers, set the query parameter as level=<desired_level>.
-  - To change a particular logger's level, set the query parameter like so, <logger_name>=<desired_level>.
+  If the default component logger is used, the logger name should be exactlly the component name.
+
+  - To change the logging level across all loggers, set the query parameter as ``level=<desired_level>``.
+  - To change a particular logger's level, set the query parameter like so, ``<logger_name>=<desired_level>``.
   - To change multiple logging levels at once, set the query parameter as ``paths=<logger_name1>:<desired_level1>,<logger_name2>:<desired_level2>``.
-  - To list the loggers, send a POST request to the /logging endpoint without a query parameter.
+  - To list the loggers, send a POST request to the ``/logging`` endpoint without a query parameter.
+
+  If ``--enable-fine-grain-logging`` is set, the logger is represented by the path of the file it belongs to (to be specific, the path determined by ``__FILE__``),
+  so the logger list will show a list of file paths, and the specific path should be used as ``<logger_name>`` to change the log level.
+
+  We also added the file basename, glob ``*`` and ``?`` support for fine-grain loggers. For example, we have the following active loggers with trace level.
```

Comment:
What happens in cases where multiple matches? e.g. `paths=???_listener_impl:info&&paths=tcp_listener_impl:warning` might be useful to document

Reply:
Done